doctor/docs/base.py

```diff
@@ -251,6 +274,51 @@ def get_json_lines(annotation: ResourceAnnotation, field: str, route: str,
                                  request)
 
 
+def get_resource_object_doc_lines() -> List[str]:
+    """Generate documentation lines for all collected resource objects.
+
+    As API documentation is generated we keep a running list of objects used
+    in request parameters and responses.  This section will generate
+    documentation for each object and provide an inline reference in the API
+    documentation.
+
+    :returns: A list of lines required to generate the documentation.
+    """
+    lines = ['Resource Objects', '----------------']
```

Comment:
Thought's on this name for the heading?  I don't hate it, but don't love it either.

Reply:
I'm also ambivalent about it, but I also can't think of anything better so I think we can just stick with this.